src/index.ts

```diff
@@ -1,70 +1,85 @@
 import debounce from 'lodash/debounce';
 import throttle from 'lodash/throttle';
 
+interface EventData {
```

Comment:
Would rather use `type` here. You only benefit from Interfaces for global types that might be extended like `window`. Whenever you're sure that you're working with a non-extendable type, go for `type`. (Pretty much: Always use `type` until you can't 😀)